pytorch_lightning/loggers/tensorboard.py

```diff
@@ -163,6 +162,10 @@ def version(self) -> int:
 
     def _get_next_version(self):
         root_dir = os.path.join(self.save_dir, self.name)
+
+        if not os.path.exists(root_dir):
```

Comment:
```suggestion
        if not os.path.isdir(root_dir):
            log.warning('Missing logger folder: %s', root_dir)
```
with adding `from pytorch_lightning import _logger as log`